### DISCUSSION TITLE
Out of focus windows don't get grayed out

### DISCUSSION THREAD

**Discussion by fulalas** [lah7/gtk3-classic#105] [Help & Support] [answered]
The expected behaviour would be like this (window at left is out of focus and its elements are grayed out, while the window at right is in focus and its elements are not grayed out):
![gtk-classic-focus-issue](https://github.com/lah7/gtk3-classic/assets/27843666/480b694d-6598-4a45-a4cd-98918ec53ad7)

I tested this on Xfce 4.18, and it seems the distro doesn't matter. Just by reverting to the original gtk+3 fixes the issue.

Might be useful for the discussion: https://stackoverflow.com/questions/40475617/widget-sensitive-gray-look-change-with-window-focus-causes-too-much-delay

**Comment by lah7** [comment, marked as answer]
Can confirm. There is a patch [appearance__disable-backdrop.patch](https://github.com/lah7/gtk3-classic/blob/master/appearance__disable-backdrop.patch) that allows you to restore the original behaviour. Simply set this environment variable:

    export GTK_BACKDROP=1

Place that in your `~/.bash_aliases`, `~/.bashrc` or `/etc/environment` file then restart/re-login for changes to take effect.

Looking at the project's history, `:backdrop` was disabled to make it behave like "classic" GTK 2 for inactive windows. Funnily enough, this makes it _consistent_ with Windows and Qt 5 too, since they don't dim controls on inactive windows.

**Comment by fulalas** [comment]
Yes, reverting this patch fixes the issue. Thanks a lot!

And, indeed, it's very ironic that you're trying to mimic a limitation of GTK2 to make it consistent with other systems and toolkits :D
